pkg/operator/controller/controller_dns_configmap.go

```diff
@@ -237,3 +249,40 @@ func coreDNSLogLevel(dns *operatorv1.DNS) string {
 	}
 	return "class error"
 }
+
+func contains(upstreams []operatorv1.Upstream, upstream operatorv1.Upstream) bool {
+	for _, anUpstream := range upstreams {
+		//if upstream.Type == operatorv1.SystemResolveConfType && anUpstream.Type == upstream.Type {
+		//	return true
+		//}
+		//if upstream.Type == operatorv1.NetworkResolverType {
+		//	if anUpstream.Type == upstream.Type {
+		//		if strings.EqualFold(anUpstream.Address, upstream.Address) {
+		//			if anUpstream.Port == upstream.Port {
+		//				return true
+		//			}
+		//		}
+		//	}
+		//}
```

Comment:
No need to keep the old code in a comment.

Reply:
oops sorry about that :/